SwiftificationTests/SequenceTypeTests.swift

```diff
@@ -39,6 +39,27 @@ class SequenceTypeTests: XCTestCase {
         let actual = array.takeFirst { $0 > 2 }
         XCTAssertEqual(expected, actual)
     }
+    
+    func testTakeWhile1() {
```

Comment:
I wasn't concerned about testing the `rethrows` changes to `uniqueBy` because I think the compiler guarantees it.